projects/emojicoin/index.js

```diff
@@ -0,0 +1,27 @@
+const { function_view, timestampToVersion } = require("../helper/chain/aptos");
+
+async function getAllMarkets(ledgerVersion) {
+  // This function will get all markets and the associated TVL in APT
+  const registry = await function_view({
+    functionStr: "0xface729284ae5729100b3a9ad7f7cc025ea09739cd6e7252aff0beb53619cafe::emojicoin_dot_fun::registry_view",
+    args: [],
+    type_arguments: [],
+    ledgerVersion,
+  })
+  return registry.total_value_locked.value
```

Comment:
@gregnazario thanks for the PR, it is mostly good. One thing I am not sure of is, how is the tvl computed here? does it include the value of the memecoin in the pools?

Reply:
@g1nt0ki total value locked is indexed onchain using a global counter that is tracked at the "registry", a global manifest for all markets with select cumulative and instantaneous statistics, including TVL. It is denominated in octas (APT subunits), and includes the value of the emojicoins locked in each market at the current market price

Reply:
hi @alnoki , yeah, I thougt so, we cant count memecoin value as tvl, for launchpads we count only the deposited gas token/stablecoin, i.e only the APT value in the pools, you can check verify this by looking up the adapters for wen.markets or bigpump

Reply:
@g1nt0ki the registry also contains a field for `total_quote_locked`, which is the amount of APT locked across all pools. Should this be reported instead?

Reply:
yes, can you update it to that

Reply:
cc @gregnazario